User-Manual.md

```diff
@@ -133,13 +134,74 @@ Addons are sets of levels provided by the community. To install and play a new a
  - Select the new item(s) in this list, which you wish to play
 
 These addons are not managed by The SuperTux Team, so please do not register your issues regarding them on the SuperTux/supertux repository.
-You *may* be able to get help by making an issue [here](https://github.com/SuperTux/addons/issues)
+You _may_ be able to get help by making an issue [here](https://github.com/SuperTux/addons/issues)
 
 By default the game searches for new addons in the SuperTux Team's addon repo [here](https://github.com/SuperTux/addons).
 If you wish to submit a new addon to the repo, please make an issue or pull request there.
-If you would rather change the addon repo which the game looks for new addons in, you can start the game with the repository-url argument specified: ```supertux2 --repository-url URL```
+If you would rather change the addon repo which the game looks for new addons in, you can start the game with the repository-url argument specified: `supertux2 --repository-url URL`
 **WARNING: THIS IS NOT RECOMMENDED BY THE SUPERTUX TEAM**
 
 ## Using The Level Editor
 
-*Level Editor documentation has not been written yet.*
+### How to Start
+You can find the Level Editor in the main menu.
+
+Once you're there, you can create a new level subset and after that you can create a new level.
+
+### Placing Tiles
+Once you have created a level, you can already start to build.
+Press the **Tileset** Button in the top right coner and chose one of many diffrent tileset categories. (For example: Snow)
+
+Select a tile that you want to use and start building.
+
+There are different ways to place tiles:
+
+ - Placing one tile at a time by pressing the left mouse button. ![](https://github.com/SuperTux/supertux/blob/master/data/images/engine/editor/select-mode0.png)
```

Comment:
Rather than linking to the master version, please link to a specific commit of the file (probably the latest as of writing). This avoids voiding the link if a filename or directory is changed.